src/tests/integrity.test.ts

```diff
@@ -41,6 +36,27 @@ describe('Normal flow', (): void => {
             .to.be.instanceOf(SequelizeUser)
             .and.to.have.property('id')
             .that.is.a('number')
-            .that.is.equal(secondUser.id);
+            .and.is.equal(secondUser.id);
+    });
+
+    it('should be able to create a user, issue a token for it and' +
+        ' authenticate the token', async (): Promise<void> => {
+        const {SequelizeAccessToken, SequelizeUser} = models;
+        const user = new SequelizeUser({username: 'a', isActive: true});
```

Comment:
In tests, my preferece is using real world example assigned to a const variable e.g. USER = 'User1!3m' or even using unicode characters if you support them. `a` and `b` can miss some of the potential bugs, etc.

Reply:
@hesamd done, in the new commit I used faker to generate test values.